src/app/components/vending-machine/modal/_product/AppVendingMachineProduct.vue

```diff
@@ -74,6 +75,25 @@ const anchorStyles = computed<CSSProperties>(() => {
 	}
 	return {};
 });
+
+const readableEndsOnStyles: CSSProperties = {
+	...StickerPackExpiryStyles,
+	zIndex: overlayTagZIndex,
+};
+
+const readableEndsOn = computed(() => {
+	const endsOn = shopProduct.value.ends_on;
+	if (!endsOn) {
+		return;
+	}
+
+	return shorthandReadableTime(endsOn, {
```

Comment:
I know this wasn't done as part of this PR, but the way to do translations properly is a bit harder. For trailingText, you don't know where the time part should be in a language. English would put it before, but others might do it differently. So you'd have to make it a function instead that gets passed the time as a var and then interpolate it in, like this:

`trailingText: (time) => $gettextInterpolate('%{ time } left to purchase', { time })`